config/buildspec.yml

```diff
@@ -1,28 +1,33 @@
-# Build Spec for AWS CodeBuild CI
+# Build Spec for AWS CodeBuild CI which will run for every PR.
 
 version: 0.2
 env:
   variables:
-    run_pytest_pytorch: "enable"
+    ## below enviornment variables are overridden in respective code build.
+    # for example for pytorch build run_pytest_pytorch and run_integration_pytest_pytorch will be enabled
+    run_pytest_pytorch: "disable"
     run_pytest_mxnet: "enable"
-    run_pytest_tensorflow: "enable"
-    run_pytest_xgboost: "enable"
-    run_integration_pytest_pytorch: "enable"
+    run_pytest_tensorflow: "disable"
+    run_pytest_tensorflow2: "disable"
+    run_pytest_xgboost: "disable"
+    run_integration_pytest_pytorch: "disable"
     run_integration_pytest_mxnet: "enable"
-    run_integration_pytest_tensorflow: "enable"
+    run_integration_pytest_tensorflow: "disable"
+    run_integration_pytest_tensorflow2: "disable"
```

Comment:
why disable?

Reply:
@anirudhacharya

Reply:
because these variables will get overriden here - https://tiny.amazon.com/c334slmv/IsenLink in the codebuild project